Makefile

```diff
@@ -128,6 +128,16 @@ chrome:
 		sudo ln -s "$$(which chromium)" $(CHROME_PATH); \
 	fi
 
+# https://sites.google.com/a/chromium.org/chromedriver/downloads/version-selection
+chromedriver: curl grep unzip chrome
+	if [[ -z "$$(which chromedriver)" ]]; then \
+		CHROME_VERSION=$$(google-chrome --version | grep -ioE "[0-9]+\.[0-9]+\.[0-9]+"); \
+		CHROMEDRIVER_VERSION=$$(curl https://chromedriver.storage.googleapis.com/LATEST_RELEASE_$${CHROME_VERSION}); \
+		sudo curl https://chromedriver.storage.googleapis.com/$${CHROMEDRIVER_VERSION}/chromedriver_linux64.zip -o $(CHROMEDRIVER_PATH).zip; \
+		sudo unzip $(CHROMEDRIVER_PATH).zip -d $$(dirname $(CHROMEDRIVER_PATH)); \
+		sudo chmod +x $(CHROMEDRIVER_PATH); \
```

Comment:
```suggestion
```

Reply:
I think we still need this; `which chromedriver` fails without it.